Add containerized example and improve readme (#17)
1Password/connect-sdk-js: example/README.md

# JavaScript Connect SDK Example

> This example has been created in order to illustrate the main available functionality with regard to item manipulation, in the JavaScript Connect SDK.

Reviewer: ```suggestion
This example demonstrates how to create, read, and modify items available to your 1Password Connect deployment using the JavaScript Connect SDK.
```

Given a (secret) string, this example will create an item, add it to given vault, retrieve it and, eventually, remove it.

## Prerequisites

In order to be able to run this example, one will need:

* [Docker](https://docs.docker.com/install/), installed and running

> * an [1Password Connect](https://support.1password.com/secrets-automation/#step-2-deploy-a-1password-connect-server) server, hosted on your infrastructure

Reviewer: ```suggestion
* a [1Password Connect](https://support.1password.com/secrets-automation/#step-2-deploy-a-1password-connect-server) server, hosted on your infrastructure
```

* a Connect token, with read/write permissions for at least one vault accessible by the Connect instance

## Running the example

Build the JavaScript docker demo
```
 docker build -t js-connect-sdk-example .
```

Run the docker demo with the required fields passed as environment variables
```

> docker run -it -e OP_CONNECT_TOKEN=<YOUR_CONNECT_TOKEN> -e OP_VAULT=<YOUR_VAULT_ID> /
>     -e OP_CONNECT_HOST=<YOUR_CONNECT_HOST> -e SECRET_STRING=<ANY_RANDOM_STRING> /
>     js-connect-sdk-example

Reviewer: ```suggestion
docker run -it \
    -e OP_CONNECT_TOKEN=<YOUR_CONNECT_TOKEN> \
    -e OP_VAULT=<YOUR_VAULT_ID> \
    -e OP_CONNECT_HOST=<YOUR_CONNECT_HOST> \ 
    -e SECRET_STRING=<ANY_RANDOM_STRING> \
    js-connect-sdk-example
```

Let's space this out so it's easier to read and modify in the future.

# JavaScript Connect SDK Example

This example demonstrates how to create, read, and modify items available to your 1Password Connect deployment using the JavaScript Connect SDK.
Given a (secret) string, this example will create an item, add it to given vault, retrieve it and, eventually, remove it.

## Prerequisites

In order to be able to run this example, one will need:

* [Docker](https://docs.docker.com/install/), installed and running
* a [1Password Connect](https://support.1password.com/secrets-automation/#step-2-deploy-a-1password-connect-server) server, hosted on your infrastructure
* a Connect token, with read/write permissions for at least one vault accessible by the Connect instance

## Running the example

Build the JavaScript docker demo
```
 docker build -t js-connect-sdk-example .
```

Run the docker demo with the required fields passed as environment variables
```

> docker run -it \
>     -e OP_CONNECT_TOKEN=<YOUR_CONNECT_TOKEN> \
>     -e OP_VAULT=<YOUR_VAULT_ID> \
>     -e OP_CONNECT_HOST=<YOUR_CONNECT_HOST> \ 
>     -e SECRET_STRING=<ANY_RANDOM_STRING> \
>     js-connect-sdk-example
> ```

Reviewer: Syntax change that I neglected earlier:

```suggestion
docker run \
--env OP_CONNECT_TOKEN=<YOUR_CONNECT_TOKEN> \
--env OP_VAULT=<YOUR_VAULT_ID> \
--env OP_CONNECT_HOST=<YOUR_CONNECT_HOST> \ 
--env SECRET_STRING=<ANY_RANDOM_STRING> \
-it js-connect-sdk-example
```
```
